src/com/lilithsthrone/game/character/effects/StatusEffect.java

```diff
@@ -5343,7 +5343,7 @@ public String extraRemovalEffects(GameCharacter target) {
 		@Override
 		public boolean isConditionsMet(GameCharacter target) {
 			float cumAmount = target.getTotalFluidInArea(SexAreaOrifice.ANUS) + target.getTotalFluidInArea(SexAreaOrifice.MOUTH) + target.getTotalFluidInArea(SexAreaOrifice.VAGINA);
-			return cumAmount >= CumProduction.SEVEN_MONSTROUS.getMaximumValue()
+			return cumAmount >= Math.pow((Math.cbrt(11356) / 182.88) * target.getHeightValue(), 3)
```

Comment:
This calculation is almost equal to `Math.pow(target.getHeightValue(), 3) / 539` (~0.07% difference) and is a much simpler calculation.